### DISCUSSION TITLE
Spawning a machine as a behavior?

### DISCUSSION THREAD

**Discussion by geekox86** [statelyai/xstate#3569] [Q&A] [answered]
Greetings all,

Is it possible to use `spawnBehavior` with a machine?

I tried it and got a type error of incompatible context with actorCtx.

I am trying to spawn actors outside assign actions and set their parents manually.

Thank you.

**Comment by davidkpiano** [comment, marked as answer]
It is not possible in v4, since they have a different interface. It will be possible in v5.
